Tests/L0/Maven/_suite.ts

```diff
@@ -121,6 +121,24 @@ function createTempDirsForSonarQubeTests(): void {
     }
 }
 
+function captureStream(stream):{unhook():void, captured():string} {
```

Comment:
This function handles the business of capturing _process.stdout_ for the duration of a test. It redirects all calls to the _write()_ function of the given stream until _unhook()_ is called. All redirected calls are stored in the internal buffer, which is exposed and can be asserted on.